adcirc_nn/coupler/adcirc_nn_class.py

```diff
@@ -118,10 +118,10 @@ def coupler_initialize(self, argc, argv):
         self.adcircntsteps=0+self.pmain.itime_end #Needed 0+ to prevent the two from being the same object :-/ Careful!!!!
         self.adcircfort19pathname=''.join(np.append(np.char.strip(self.ps.inputdir),'/fort.19.new'))
         self.adcircedgestringid=int(argv[argc.value-1])-1
-        self.adcircedgestringnnodes=self.pb.nvell[self.adcircedgestringid]
-        self.adcircedgestringnodes=self.pb.nbvv[1:self.adcircedgestringnnodes]
+        self.adcircedgestringnnodes=self.pb.nvdll[self.adcircedgestringid]
```

Comment:
Is this just a typo/minor fix or does it have anything to do with changing boundary condition?

Reply:
`nvell` is an ADCIRC variable related to for flux boundary condition, whereas `nvdll` is one related to elevation boundary condition.